docs/ja/config/README.md

```diff
@@ -0,0 +1,615 @@
+# 設定
+
+::: tip 🔥Starshipの開発は現在も進んでいます。多くの新しいオプションが今後のリリースで利用可能になります。 :::
+
+starshipの設定を開始するには、`~/.config/starship.toml` ファイルを作成します。
+
+```shell
+$ touch ~/.config/starship.toml
+```
+
+starshipのすべての設定は、この[TOML](https://github.com/toml-lang/toml)ファイルで行われます。
+
+```toml
+# プロンプト表示時に改行しない
+add_newline = false
+
+# "❯" から "➜" に表示を変更
+[character]      # The name of the module we are confguring is "character"
+symbol = "➜"     # The "symbol" segment is being set to "➜"
+
+# packageモジュールを無効にする
+[package]
+disabled = true
+```
+
+### 用語
+
+**モジュール**: OSのコンテキスト情報に基づいて情報を提供するプロンプト内のコンポーネントです。たとえば、現在のディレクトリがNodeJSプロジェクトである場合、「nodejs」モジュールは、現在コンピューターにインストールされているNodeJSのバージョンを表示します。
+
+**セグメント**: モジュールを構成する小さなサブコンポーネントです。たとえば、「nodejs」モジュールの「symbol」セグメントには、バージョン番号の前に表示される文字が含まれています（デフォルト: ⬢）。
+
+以下はNode モジュールの表現です。次の例では、「シンボル」と「バージョン」はその中のセグメントです。すべてのモジュールには、デフォルトの端末色であるprefixとsuffixもあります。
+
+```
+[prefix]      [symbol]     [version]    [suffix]
+ "via "         "⬢"        "v10.4.1"       ""
+```
+
+### スタイルの設定
+
+starshipのほとんどのモジュールでは、表示スタイルを設定できます。これは、設定を指定する文字列であるエントリ（`style`）で行われます。スタイル文字列の例とその機能を次に示します。完全な構文の詳細については、詳細は [高度な設定](/advanced-config/)を参照してください 。
+
+- `"fg:green bg:blue"`は、青色の背景に緑色のテキストを設定します
+- `"bg:blue fg:bright-green"`は、青色の背景に明るい緑色のテキストを設定します
+- `"bold fg:27"`は、 [ANSIカラー](https://i.stack.imgur.com/KTSQa.png) 27の太字テキストを設定します
+- `"underline bg:#bf5700"`は、焦げたオレンジ色の背景に下線付きのテキストを設定します
+- `""`はすべてのスタイルを明示的に無効にします
+
+スタイリングがどのように見えるかは、端末エミュレータによって制御されることに注意してください。たとえば、一部の端末エミュレータはテキストを太字にする代わりに色を明るくします。また、一部のカラーテーマは通常の色と明るい色と同じ値を使用します。
+
+## プロンプト
+
+これは、プロンプト全体のオプションのリストです。
+
+### オプション
+
+変数 | デフォルト | 説明
+--- | --- | ---
+`add_newline` | `true` | プロンプトの開始前に新しい行を追加します。
+`prompt_order` | [link](#%E3%83%87%E3%83%95%E3%82%A9%E3%83%AB%E3%83%88%E3%81%AE%E3%83%97%E3%83%AD%E3%83%B3%E3%83%97%E3%83%88%E9%A0%86) | プロンプトモジュールを出力する順序を設定します。
+
+### 設定例
+
+```toml
+# ~/.config/starship.toml
+
+# プロンプト表示の改行を無効にする
+add_newline = false
+# デフォルトのプロンプト表示順を書き換える
+prompt_order=["rust","line_break","package","line_break","character"]
+```
+
+### デフォルトのプロンプト表示順
+
+`default_prompt_order`オプションは、空または`prompt_order`が指定されていない場合に、プロンプトにモジュールが表示される順序を定義するために使用されます。デフォルトは次のとおりです。
+
+```
+default_prompt_order = [
+    "username",
+    "hostname",
+    "directory",
+    "git_branch",
+    "git_state",
+    "git_status",
+    "package",
+    "nodejs",
+    "ruby",
+    "rust",
+    "python",
+    "golang",
+    "nix_shell",
+    "cmd_duration",
+    "line_break",
+    "jobs",
+    "time",
+    "battery",
+    "character",
+]
+```
+
+## バッテリー
+
+`battery`モジュールは、デバイスのバッテリー残量と現在の充電状態を示します。モジュールは、デバイスのバッテリー残量が10％未満の場合にのみ表示されます。
```

Comment:
@heyrict Thanks for your comment on https://gitlocalize.com/repo/3018/ja/review/10412.

I fixed it 👍